glean-core/src/histogram/exponential.rs

```diff
@@ -51,142 +46,60 @@ fn exponential_range(min: u64, max: u64, bucket_count: usize) -> Vec<u64> {
     ranges
 }
 
-/// The type of a histogram.
-#[derive(Copy, Clone, Debug, Serialize, Deserialize)]
-#[serde(rename_all = "lowercase")]
-pub enum Type {
-    Exponential,
-}
-
-/// A histogram.
-///
-/// Stores the ranges of buckets as well as counts per buckets.
-/// It tracks the count of added values and the total sum.
-///
-/// ## Example
-///
-/// ```rust,ignore
-/// let mut hist = Histogram::exponential(1, 500, 10);
-///
-/// for i in 1..=10 {
-///     hist.accumulate(i);
-/// }
-///
-/// assert_eq!(10, hist.count());
-/// assert_eq!(55, hist.sum());
-/// ```
 #[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct Histogram {
-    min: u64,
-    max: u64,
+pub struct Exponential {
```

Comment:
This could be confusing since `Functional` histograms are technically `Exponential` as well.  Maybe `Functional` vs. `Precomputed` instead?

Reply:
But then "linear" is also a precomputed histogram, but the computation function is different.

Reply:
Maybe `PrecomputedExponential`?

Reply:
`PrecomputedExponential`++